tests/e2e_logging.py

```diff
@@ -302,7 +302,7 @@ def test_historical_query(network, args):
 
 
 @reqs.description("Testing forwarding on member and node frontends")
-@reqs.supports_methods("/node/tx")
+@reqs.supports_methods("/node/tx/{view}/{seqno}")
```

Comment:
Should we assume that this endpoint is offered by all CCF services?

Reply:
It is!